cern_access/tests/util_test.py

```diff
@@ -26,3 +34,67 @@ def test_license_plate_handling_error():
     assert sanitize_license_plate('------') is None
     assert sanitize_license_plate('123/456') is None
     assert sanitize_license_plate('VALID 1234 \U0001F4A9') is None
+
+
+@pytest.mark.parametrize('mock_access_request',
+                         [{
```

Comment:
Ugh wtf is this indentation? seems to be consistent with what there was before, but still, wtf...

how about this?

```python
@pytest.mark.parametrize('mock_access_request', [{
    'during_registration': True,
    'during_registration_required': False,
    'personal_data': generate_personal_data(True),
    'include_accompanying_persons': True,
}], indirect=True)
```